docs/docs/contributing-guide/setup/macos.md

```diff
@@ -32,8 +32,20 @@ Follow these steps to setup and run ToolJet on macOS for development purposes. O
     ```bash
     brew install postgresql
     ```
-
-    1.4 Clone the repository
+    
+    1.4 Install PostgREST(optional)
+    
+    :::info 
+    Required only if TooljetDb is being used. 
+
+    Please use PostgREST version 10.1.1.x
+    :::       
+    
+    ```bash
+    brew install postgrest
```

Comment:
Please check command to specify the version to install here

Reply:
adding the command with version